manifests/cf-manifest/deployments/aws/060-aws-broker-stub.yml

```diff
@@ -0,0 +1,18 @@
+resource_pools:
+  - name: rds_broker_z1
+    network: cf1
+    stemcell: (( grab meta.stemcell ))
+    env: (( grab meta.default_env ))
+    cloud_properties:
+      instance_type: m3.medium
+      ephemeral_disk:
+        size: 10240
+        type: gp2
+      availability_zone: (( grab meta.zones.z1 ))
+      iam_instance_profile: cf-cloudcontroller
```

Comment:
It should. However, this story wasnt about getting the broker working, to provision rds dbs, it was to get the broker listed in the marketplace - we felt that sorting out the IAM roles / permissions belonged in the story about getting DB provisioning working.